cmd/snap-preseed/main_test.go

```diff
@@ -317,6 +317,7 @@ type Fake16Seed struct {
 	Essential         []*seed.Snap
 	LoadMetaErr       error
 	LoadAssertionsErr error
+	IterErr           error
```

Comment:
Should we have a test where this is set to a real error? It seems to me that this field is never assigned to, so Fake16Seed.Iter() is effectively always returning `null`.

Maybe this is intended, just asking :-)

Reply:
Hm Iter() doesn't return errors on its own, perhaps we're missing some mocking in preseeding tests here.

Reply:
if its never used in these tests and code  it's better to not define it